File: VB/WindowsFormsApplication40/MyGalleryControlGallery.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraBars.Ribbon.Gallery
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraBars.Ribbon.ViewInfo

Namespace WindowsFormsApplication40
	Friend Class MyGalleryControlGallery
		Inherits GalleryControlGallery
		Public Sub New()

		End Sub
		Public Sub New(ByVal galleryControl As DevExpress.XtraBars.Ribbon.GalleryControl)
			MyBase.New(galleryControl)

		End Sub
		Public ReadOnly Property MyViewInfo() As StandaloneGalleryViewInfo
			Get
				Return Me.ViewInfo
			End Get
		End Property
	End Class
End Namespace

File: CS/WindowsFormsApplication40/MyGalleryControlGallery.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraBars.Ribbon.Gallery;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraBars.Ribbon.ViewInfo;

namespace WindowsFormsApplication40
{
    class MyGalleryControlGallery : GalleryControlGallery
    { 
        public MyGalleryControlGallery()
        {

        }
        public MyGalleryControlGallery(DevExpress.XtraBars.Ribbon.GalleryControl galleryControl)
            : base(galleryControl)
        {
            
        }
        public StandaloneGalleryViewInfo MyViewInfo { get { return this.ViewInfo; } }
    }
}
